benchbuild/projects/gentoo/xz.py

```diff
@@ -29,23 +29,20 @@ def compile(self):
         download.Wget(test_url, test_archive)
         tar("fxz", test_archive)
 
-    def run_tests(self, runner):
+    def run_tests(self):
         xz = wrapping.wrap(local.path("/usr/bin/xz"), self)
+        xz = run.watch(xz)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable name "xz" doesn't conform to '[a-z_][a-z0-9_]{2,30}$' pattern](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5028678)